index.js

```diff
@@ -52,5 +54,5 @@ assets.load({
   BASE: '../assets/img/base.png',
   TOWER: '../assets/img/tower.png'
 }).then(() => {
-  document.body.appendChild(app.start())
+  document.body.appendChild(app.start({hash: true}))
```

Comment:
this should technically enable us to go directly to the tutorial levels, but doesn't...